src/pages/studyView/addChartButton/AddChartButton.tsx

```diff
@@ -525,19 +527,18 @@ class AddChartTabs extends React.Component<IAddChartTabsProps, {}> {
                             this.selectedGenericAssayProfileIdByType.get(type)
                     )?.profileIds || [];
 
-                const entitityMap = molecularProfileIdsInType.reduce(
-                    (acc, profileId) => {
-                        this.props.store.genericAssayEntitiesGroupedByProfileId.result![
-                            profileId
-                        ].forEach(meta => {
-                            acc[meta.stableId] = meta;
-                        });
-                        return acc;
-                    },
-                    {} as { [stableId: string]: GenericAssayMeta }
-                );
+                const entityMap: {
```

Comment:
i'm pretty sure ts can imply the type here if you specify it on the acc argument of the reduce callback.  
```reduce((acc:TYPE, profileId)=> ... ```
that way you don't need it at ln 530 or 539